Exercise 3/styles.css

```diff
@@ -13,19 +20,29 @@
 
 .box-1 {
     background-color: #f1e0c5;
+    grid-column: 1 / -1;
 }
 .box-2 {
     background-color: #c9b79c;
+    grid-column: 2 / -2;
 }
 .box-3 {
     background-color: #71816d;
+    grid-row: 2 / -2;
+    grid-column: 1 / span 1;
 }
 .box-4 {
     background-color: #342a21;
+    grid-row: 3 / span 1;
+    grid-column: 2 / -2;
 }
 .box-5 {
     background-color: #da667b;
+    grid-row: 2 / -2;
+    grid-column: -1 / -2;
```

Comment:
Try using `span` instead of these negative values. Using `-1` is fine for getting the last element when the number of items is unknown, but otherwise it gets quite unclear.